nilearn/_utils/niimg_conversions.py

```diff
@@ -33,6 +33,23 @@ def _check_same_fov(img1, img2):
             and np.allclose(img1.get_affine(), img2.get_affine()))
 
 
+def _assert_same_fov(**kwargs):
```

Comment:
It is a bit of a shame that _assert_same_fov doesn't reuse _check_same_fov given their similar functionality.